### PR TITLE
monai env vars

### PR DESCRIPTION
**Pull request by rijobro**
### Description
MONAI environment variable wrapper. Avoids raw strings polluting MONAI code.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] New tests added to cover the changes.

### CODE REVIEW

File: tests/test_monai_env_vars [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class TestMONAIEnvVars(unittest.TestCase):
+    def test_monai_env_vars(self):
+        os.environ["MONAI_DEBUG"] = "42"
+        self.assertEqual(os.environ.get("MONAI_DEBUG"), "42")
+        self.assertEqual(MONAIEnvVars.debug(), "42")
+
+
+
+if __name__ == "__main__":
+    unittest.main()
```
Comment (wyli) [inline tests/test_monai_env_vars:26]: missing `.py` in the file names `test_monai_env_vars` and `test_trasnform`

File: monai/utils/misc.py [hunk 2]
```diff
@@ -360,6 +360,52 @@ def copy_to_device(
...
+    false_set = {"no", "false", "f", "n", "0"}
+    if isinstance(value, str):
+        value = value.lower()
+        if value in true_set:
+            return True
+        if value in false_set:
+            return False
+
+    if raise_exc:
+        raise ValueError('Expected "%s"' % '", "'.join(true_set | false_set))
```
Comment (ericspod) [inline monai/utils/misc.py:386]:
```suggestion
        raise ValueError(f"Expected a value from: {', '.join(map(repr, true_set | false_set))}")
```

Just a little more explanatory

**Review by ericspod — APPROVED** [review]
Other than the file names looks good, my one comment was just a suggestion.

**Comment by rijobro** [comment]
@wyli @ericspod thanks both!

**Pull request merged**